celestia_test.go

```diff
@@ -45,7 +45,7 @@ func (t *TestSuite) SetupSuite() {
 	}
 
 	// pulls an image, creates a container based on it and runs it
-	resource, err := pool.Run("ghcr.io/rollkit/local-celestia-devnet", "4ecd750", []string{})
+	resource, err := pool.Run("ghcr.io/rollkit/local-celestia-devnet", "394c0b8", []string{})
```

Comment:
it'd be neat to move this to a const at the top of the _test.go file so its easy to source/find and suggests that its being used in the test

```
const(
  localDevnetImageVersion = "394c0b8"
)
```

or something?

Reply:
Sounds good 👍🏽